examples/02_meta-analyses/plot_simulated_data.py

```diff
@@ -0,0 +1,228 @@
+# Create function to perform a meta-analysis and plot results
+# -----------------------------------------------------------
+
+
+def analyze_and_plot(dset, ground_truth_foci=None, return_cres=False):
+    mkda = nimare.meta.mkda.MKDADensity(kernel__r=10)
+    mkda.fit(dset)
+    corr = nimare.correct.FWECorrector(method="montecarlo", n_iters=100, n_cores=1)
+    cres = corr.transform(mkda.results)
+
+    # get the z coordinates
+    if ground_truth_foci:
+        stat_map_kwargs = {
+            "cut_coords": [c[2] for c in ground_truth_foci],
+        }
+    else:
+        stat_map_kwargs = {}
+
+    display = plot_stat_map(
+        cres.get_map("logp_level-voxel_corr-FWE_method-montecarlo"),
+        display_mode="z",
+        draw_cross=False,
+        cmap="Purples",
+        threshold=1.0,
+        vmax=15,
+        symmetric_cbar=False,
+        **stat_map_kwargs,
+    )
+
+    if ground_truth_foci:
+        # place red dots indicating the ground truth foci
+        display.add_markers(ground_truth_foci)
+
+    if return_cres:
+        return display, cres
+    return display
+
+
+###############################################################################
+# Create Dataset
+# --------------------------------------------------
+# In this example, each of the 30 generated fake studies
+# select 4 coordinates from a probability map representing the probability
+# that particular coordinate will be chosen.
+# There are 4 "hot" spots centered on 3D gaussian distributions,
+# meaning each study will likely select 4 foci that are close
+# to those hot spots, but there is still random jittering.
+# Each study has a mean ``sample_size`` of 30 with a variance of 10
+# so some studies may have fewer than 30 participants and some
+# more.
+
+ground_truth_foci, dset = create_coordinate_dataset(
+    n_foci=4, fwhm=10.0, sample_size=30, sample_size_interval=10, n_studies=30
+)
+
+###############################################################################
+# Analyze and plot simple dataset
+# -------------------------------
+
+analyze_and_plot(dset, ground_truth_foci)
+
+###############################################################################
+# Fine-tune dataset creation
+# --------------------------------------------------
+# Perhaps you want more control over the studies being generated.
+# You can manually specify how many "converging" foci you want specified
+# in each study (i.e., ``n_foci``).
+# This way you can test what happens if one study has way more reported
+# foci than another study.
+# You can also:
+#   - manually specify the sample sizes for each study (i.e., ``sample_size``).
+#   - set the number of false positive foci for each study (i.e., ``n_noise_foci``).
+#   - set the weighed probability of each foci being chosen (i.e., ``foci_weights``).
+#   - set the dispersion of the gaussian spread for each foci (i.e., ``fwhm``).
+#   - set the specific ground truth foci (i.e., ``foci_coords``).
+
+num_studies = 30
```

Comment:
I think this was just missed in the recent set of changes.

Reply:
fixed!